src/packages/@ncigdc/modern_components/ClinicalAnalysis/ContinuousAggregationQuery.js

```diff
@@ -20,22 +20,52 @@ const simpleAggCache = {};
 const pendingAggCache = {};
 const DEFAULT_CONTINUOUS_BUCKETS = 4;
 
-const getContinuousAggs = ({
-  fieldName, stats, filters,
-}) => {
+const getContinuousAggs = ({ fieldName, stats, filters, bins }) => {
   // prevent query failing if interval will equal 0
   if (_.isNull(stats.min) || _.isNull(stats.max)) {
     return null;
   }
-  const interval = (stats.max - stats.min) / DEFAULT_CONTINUOUS_BUCKETS;
+
+  let rangeArr = _.reduce(bins, (acc, bin, key) => {
+    if (
+      bin.from &&
```

Comment:
`from` or `to` could be 0 which would make this condition fail

Reply:
Just find out you put comment here. Yes. You are right. I will correct it.